db/migrate/20151123235600_add_purchase_type_to_gsa18f_procurements.rb

```diff
@@ -0,0 +1,11 @@
+class AddPurchaseTypeToGsa18fProcurements < ActiveRecord::Migration
+  def change
+    add_column :gsa18f_procurements, :purchase_type, :integer
```

Comment:
Thoughts about using Postgres's [`enum`](http://www.postgresql.org/docs/9.3/static/datatype-enum.html) data type, rather than an integer? Seems like it might make querying a bit more straightforward (since it would use the string rather than an arbitrary integer). Also makes it easier to avoid the issue of the names/numbers accidentally getting changed.